sdk/core/core-rest-pipeline/samples/v1/javascript/README.md

```diff
@@ -0,0 +1,59 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+urlFragment: core-rest-pipeline-javascript
+---
+
+# Azure SDK Core client library samples for JavaScript
+
+These sample programs show how to use the JavaScript client libraries for Azure SDK Core in some common scenarios.
+
+| **File Name**                 | **Description**                                                         |
+| ----------------------------- | ----------------------------------------------------------------------- |
+| [node-sample.js][node-sample] | This samples shows how to make a GET request using default http client. |
+
+## Prerequisites
+
+The sample programs are compatible with Node.js >=12.0.0.
+
+You need [an Azure subscription][freesub] to run these sample programs.
```

Comment:
Azure sub is not needed for this sample.

Reply:
@witemple-msft perhaps it is a good idea to have a flag in dev-tool's publish command for non-azure-specific things?